external/js/cothority/src/log.ts

```diff
@@ -7,18 +7,11 @@ const lvlStr = ["E ", "W ", "I ", "!4", "!3", "!2", "!1", "P ", " 1", " 2", " 3"
 
 export class Logger {
 
-    set lvl(l) {
-        this._lvl = l;
-    }
-
-    get lvl() {
-        return this._lvl;
-    }
-    _lvl: number;
+    lvl: number;
     stackFrameOffset: number = 0;
 
     constructor(lvl: number) {
-        this._lvl = lvl === undefined ? defaultLvl : lvl;
+        this.lvl = lvl === undefined ? defaultLvl : lvl;
```

Comment:
Typewise, `lvl` couldn't be `undefined`, so `defaultLvl` will never be used as value.